testapi.pm

```diff
@@ -390,8 +390,9 @@ sub upload_logs {
     my ($file) = @_;
 
     bmwqemu::log_call('upload_logs', file => $file);
-    my $cmd      = "curl --form upload=\@$file ";
     my $basename = basename($file);
+    my $upname   = ref($autotest::current_test) . '-' . $basename;
```

Comment:
Agreed, even better. Where can we fail? i.e. where are all modules referenced by shortname?